packages/babel-traverse/src/scope/lib/renamer.js

```diff
@@ -105,7 +105,7 @@ export default class Renamer {
     let { binding, oldName, newName } = this;
     let { scope, path } = binding;
 
-    let parentDeclar = path.find((path) => path.isDeclaration() || path.isFunctionExpression());
+    let parentDeclar = path.find((path) => path.isDeclaration());
```

Comment:
I don't think that works anyway, since it's marked with a TODO and a `return`, but valid point. I can change how we're doing this to use two different parent declarations if you'd prefer. Let me know.